lib/lita/adapters/slack.rb

```diff
@@ -5,12 +5,21 @@ module Adapters
     class Slack < Adapter
       # Required configuration attributes.
       config :token, type: String, required: true
+      config :endpoint, type: String, required: false
+      config :proxy, type: String, required: false
```

Comment:
`required: false` is the default so this can be left off of both config attributes. Let's add the default value (https://slack.com/api) here so it doesn't need to be assigned in `#initialize`.